diff --git a/lalburst.pc.in b/lalburst.pc.in
index 118a145ff9..b254640b0c 100644
--- a/lalburst.pc.in
+++ b/lalburst.pc.in
@@ -12,6 +12,6 @@ LALBURST_PYTHON_PATH=@pyexecdir@
 Name: LALBurst
 Description: LAL Burst Library Support
 Version: @VERSION@
-Requires: gsl, lal >= @LAL_VERSION@, libmetaio, lalmetaio >= @LALMETAIO_VERSION@, lalsimulation >= @LALSIMULATION_VERSION@
+Requires.private: gsl, lal >= @LAL_VERSION@, libmetaio, lalmetaio >= @LALMETAIO_VERSION@, lalsimulation >= @LALSIMULATION_VERSION@
 Libs: -L${libdir} -llalburst
 Cflags: -I${includedir}
